src/pages/suggestions/new/index.tsx

```diff
@@ -23,13 +53,25 @@ const NewSuggestion = () => {
           onClick={() => router.push("/suggestions")}
         />
       </div>
+      {showSpinner && (
+        <div className={"h-[75vh] text-center"} data-cy="suggestions-spinner">
+          <Spinner size={10} />
+        </div>
+      )}
 
-      <div className={"mx-auto text-center lg:w-2/4"}>
-        <ProgressBar />
-        {activeArea == Area.SongInformation && <SongInformationArea />}
-        {activeArea == Area.Instruments && <InstrumentsArea />}
-        {activeArea == Area.Review && <ReviewArea />}
-      </div>
+      {showLoadingError && (
+        <div className={"mt-6"} data-cy="failed-fetching-suggestions">
+          <ErrorPopup text={"Failed to load suggestions."} closePopup={() => {}} />
```

Comment:
I don't feel like this is an appropriate text for this since you're not loading suggestions but instruments right?

Reply:
I recommend giving a clearer description of what is happening to the user and describe what they should do.

Reply:
I was unsure what would be better for the user, maybe "Failed to receive instruments. Try again or contact support."? or something similarly worded with a better description than support.

Reply:
Hmmmm, maybe something like this?
![image](https://github.com/ASVGay/the-rhapsodies/assets/54884540/d4328347-cc1a-4de2-89c9-9fab1ab9d15b)

But with other text. I think being generic might be the way. 

“Something went wrong”
You can try again. Contact support if this error persists. 

En dan bij show details misschien de error code uit de code zodat we het kunnen debuggen? Wat denk jij?

Reply:
I think the generic option is better. The error occurs at the start of the page; before the user has seen anything related to the list of instruments. And for them the only actions they can take is try again or contact support.

Reply:
I agree

Reply:
Added and implemented other feedback as well.